rowen-lab/routes/routes.js

```diff
@@ -0,0 +1,44 @@
+require("dotenv").config();
+
+const express = require('express');
+const fs = require('fs');
+const Router = module.exports = express.Router();
+
+var multer  = require('multer');
+const uploadDirectory = `${__dirname}/../uploads/`;
+
+var upload = multer({ dest: uploadDirectory });
+
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3();
+
+Router.post('/resource', upload.single(`labfile`) ,(req, res, next) => {
+  const base64data = new Buffer(req.file.destination, 'binary');
```

Comment:
Probably better to have all of this business logic factored out into a module or middleware instead of being embedded in the route.

Route should only be concerned with I/O, never anything logical.  Think of them as mini-controllers